### DISCUSSION TITLE
Can helloSystem run macOS apps?

### DISCUSSION THREAD

**Discussion by muhammadbahaa2001** [helloSystem/hello#329] [Q&A]
I am interested in helloSystem and I saw in some video on YouTube the apps are made in .app extension and the system natively can run it and mainly .app extension is for macOS (.exe for Windows, .apk for Android and .ipa for iOS/iPadOS).
So this means in the future the system
could be fully compatible with all mac apps (As real macOS)?
![Screenshot_20220125-143137](https://user-images.githubusercontent.com/76573761/150979772-40eec8be-f757-4e5b-9a96-4d7af70e203c.png)

**Comment by probonopd** [comment]
At this point helloSystem cannot run macOS apps. Maybe it will be possible in the future in [airyxOS](http://airyx.org/).
